src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ILoggerWithColor.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal interface ILoggerWithColor : ILogger
+    {
+        void LogMinimalWithColor(string data, ConsoleColor color);
```

Comment:
That is a good point. But LogMinimalWithColor does more than adding color. It uses Console.Write rather than Console.WriteLine. If that would not cause confusion, I can overload the LogMinimal method